feedback.md

```
@@ -1,310 +1,225 @@
-mkdir -p java/lib/transit/java
-curl -sO "https://raw.githubusercontent.com/sabeeirsharrma/transit/main/packages/transit-java-runtime/src/main/java/transit/java/BinaryProtocol.java" --output-dir java/lib/transit/java/
-curl -sO "https://raw.githubusercontent.com/sabeeirsharrma/transit/main/packages/transit-java-runtime/src/main/java/transit/java/TransitServer.java" --output-dir java/lib/transit/java/
-curl -sO "https://raw.githubusercontent.com/sabeeirsharrma/transit/main/packages/transit-java-runtime/src/main/java/transit/java/TransitService.java" --output-dir java/lib/transit/java/
-
-# Compile Java runtime
-mkdir -p java/build
-javac -d java/build java/lib/transit/java/*.java
-
-# Compile user code against the runtime
-javac -cp java/build -d java/build java/src/main/java/com/demo/*.java
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)